src/tenable/load_vulnerabilities.py

```diff
@@ -93,10 +93,10 @@
 """
 
 
-def load_tenable_vulnerabilities(export: VulnExportStatus, to: mariadb.Connection):
+def load_tenable_vulnerabilities(export: VulnExportStatus):
     """Loads all vulnerabilities within the given export to the database"""
 
-    conn = to
+    conn = mariadb.connect(**CONN_PARAMS)
```

Comment:
with the `ConnectionPool` parameter, you can obtain a `Connection` from the pool